db/migrate/20160825213523_change_load_balancer_listener_port_to_port_range.rb

```diff
@@ -0,0 +1,55 @@
+class ChangeLoadBalancerListenerPortToPortRange < ActiveRecord::Migration[5.0]
+  def change
+    reversible do |dir|
+      dir.up do
+        # convert load_balancer_port and instance_port into ranges, preserving
+        # existing values
+        execute <<-SQL
+          ALTER TABLE load_balancer_listeners
+            ALTER COLUMN load_balancer_port SET DATA TYPE int4range
```

Comment:
I assume that we have already created the new column  `instance_port_range` and we need to convert data from 
`instance_port` to `instance_port_range`

so it can be for up direction :

```
LoadBalancerListener.where.not(:instance_port => nil ).each do |x|
   x.update(:instance_port_range => x.instance_port..x.instance_port)
end
```
...

is it works for you ?